model/src/test/kotlin/config/OrtConfigurationTest.kt

```diff
@@ -39,10 +36,8 @@ import org.ossreviewtoolkit.utils.test.shouldNotBeNull
 class OrtConfigurationTest : WordSpec({
     "OrtConfiguration" should {
         "be deserializable from HOCON" {
-            val hocon = javaClass.getResource("/reference.conf").readText()
-
-            val config = ConfigFactory.parseString(hocon)
-            val ortConfig = config.extract<OrtConfiguration>("ort")
+            val refConfig = File("src/main/resources/reference.conf")
```

Comment:
To load a configuration via the OrtConfiguration class a File is required. I found it easier to provide the relative path to the file than trying to convert a URL to a File.
The original test was loading the reference configuration file via the TypeSafe Config API, which accepts URLs directly. This was a bit cheating IMHO because it was circumventing the OrtConfiguration.load() function, which should actually be tested.